evoks/Term/models.py

```
@@ -24,42 +24,53 @@ def create(cls, name: str, uri: str):
+    def export_term(self, dataformat: str) -> dict:
+        """Exports a single term in the requested RDF format.
 
-    def export_term(self, data_format: str) -> None:
+        Args:
+            dataformat (str): Desired data format (json-ld, turtle, rdf+xml).
+
+        Returns:
+            dict: Dictionary containing file content, content type, and content disposition.
+        """
         from evoks.fuseki import fuseki_dev
+        from rdflib import Graph
```

Ruff (0.8.2)

`rdflib.Graph` imported but unused

Remove unused import: `rdflib.Graph`

(F401)